.travis.yml

```diff
@@ -10,18 +9,23 @@ before_script:
   - sleep 3
 
 install:
-  - pip uninstall -y pytest
-  - pip install -r requirements.txt colorama
+  - pip install -r requirements.txt -U pytest==$PYTEST_VERSION
+
+jobs:
+  include:
+    - env: RUN_COMMAND="python run.py" PYTEST_VERSION=3.7.3 PLUGINCOMPAT_SITE=http://plugincompat.herokuapp.com
+      python: '2.7'
+    - env: RUN_COMMAND="python run.py" PYTEST_VERSION=3.7.3 PLUGINCOMPAT_SITE=http://plugincompat.herokuapp.com
+      python: '3.6'
+    - env: RUN_COMMAND="pytest test_web.py"
```

Comment:
@wimglenn this is the main change I did, I think it makes sense to always run `test_web.py` and the `run.py` job, even on PRs.

Reply:
But then run.py will execute even if it has bad changes.  Also I thought the secret is not supposed to be available in PRs?

Reply:
They are available for PRs in the main repository, but not on forks. #29 failed because of that. 😉

Reply:
Ah, OK!